BDArmory/Bullets/PooledBullet.cs

```diff
@@ -918,7 +977,7 @@ bool BulletHitAnalysis(RaycastHit hit, float period)
                 bool viableBullet = ProjectileUtils.CalculateBulletStatus(bulletMass, caliber, sabot);
 
                 ProjectileUtils.StealResources(hitPart, sourceVessel, stealResources);
-                ProjectileUtils.CheckPartForExplosion(hitPart);
+                //ProjectileUtils.CheckPartForExplosion(hitPart);
```

Comment:
This was the cause of fueltanks occasionally randomly exploding when shot, old legacy code